psydac/linalg/utilities.py

```diff
@@ -84,6 +84,11 @@ def petsc_to_psydac(x, Xh):
     x : PETSc.Vec
       PETSc vector
 
+    Xh: psydac.linalg.basic.VectorSpace | psydac.linalg.block.BlockVectorSpace
+      Space of the coefficients of the Psydac vector.
```

Comment:
`VectorSpace` is the base class of `StencilVectorSpace` and `BlockVectorSpace`. I believe that you wanted to write as for the return value `u`:
```suggestion
    Xh: psydac.linalg.stencil.StencilVector | psydac.linalg.block.BlockVectorSpace
      Space of the coefficients of the Psydac vector.
```

Reply:
I almost agree. I think it should be `psydac.linalg.stencil.StencilVectorSpace`

Reply:
Just corrected 😉